### DISCUSSION TITLE
Leap motions not assigned per default

### DISCUSSION THREAD

**Discussion by StephanRoemer** [LazyVim/LazyVim#502] [Q&A] [answered]
I noticed today that Leap motions aren’t assigned per default.
This is very helpful when pressing “ds” and then the next 2 characters in order to “delete til”.
Would it be possible to consider a default assignment, since I think this would come handy for other users as well?

**Comment by dpetka2001** [comment, marked as answer]
I'm using default configuration and what you're describing is possible to do. I press `ds` and then 2 more characters to delete until. Maybe it's some custom configuration that messes up what you want to do?

**Comment by StephanRoemer** [comment]
Damn, you are right. I replaced mini-surround with nvim-surround because I liked it more and I totally forgot about that.
And now it stole the default shortcuts...
